package.json

```diff
@@ -12,7 +12,7 @@
     "url": "https://github.com/atom/atom/issues"
   },
   "license": "MIT",
-  "electronVersion": "3.1.10",
+  "electronVersion": "4.2.5",
```

Comment:
4.2.6 is also available

Reply:
and 4.2.7

Reply:
@rafeca I guess the 4.2.7 assets were not live yet when you tried to push the update

Reply:
Yup, I tried 4.2.7 and it didn't work well (the release notes were not even published). I'm gonna upgrade to 4.2.7 now